cpp/CMakeLists.txt

```diff
@@ -136,7 +136,7 @@ include(ConfigureNvgraph)
 
 ###################################################################################################
 # - Find and add different modules and supporting repos -------------------------------------------
-find_package(Boost 1.45.0 COMPONENTS system)
+find_package(Boost REQUIRED)
```

Comment:
@afender would the change of not getting only the system COMPONENTS affect anything negatively? Also I believe the change removing 1.45 is just removing the minimum version cmake looks for

Reply:
Considering how close to the release we are, I'm fine as long as build and test are passing.